File: Top 100 Codes/01-Getting Started/02-Check Whether a Number is Even or Odd/CheckWhetheraNumberisEvenorOdd.cpp
#include <iostream> // Include the input-output stream library

using namespace std; // Use the standard namespace to simplify syntax

int main() {
    int number; // Variable to store user input

    // Prompt the user to enter a number
    cout << "Enter a number: ";
    cin >> number; // Take user input

    // Check if the number is divisible by 2 using the modulo operator
    // If the remainder when divided by 2 is 0, the number is even; otherwise, it is odd
    number % 2 == 0 ? cout << "Even" : cout << "Odd";

    return 0; // Indicate successful execution
}

File: Top 100 Codes/01-Getting Started/02-Check Whether a Number is Even or Odd/CheckWhetheraNumberisEvenorOdd.py
# Prompt the user to enter a number
num = int(input("Enter a number: "))  # Convert the input to an integer

# Use a single-line conditional expression to check if the number is even or odd
# If the number is divisible by 2 (remainder is 0), print "Even"
# Otherwise, print "Odd"
print("Even") if num % 2 == 0 else print("Odd")
